python/minpy/__init__.py

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env python
```

Comment:
I think the eventual structure should be something like
mxnet
  autograd
    autograd
    optim
  nn
  rnn